### PR TITLE
Add polygon2 debian

### PR DESCRIPTION
**Pull request by Athanaseus**

### CODE REVIEW

File: debian/files [hunk 1]
```diff
@@ -0,0 +1,3 @@
+polygon2-dbgsym_2.0.8-1kern1_amd64.ddeb debug optional
```
Comment (gijzelaerr) [inline debian/files]: you should not commit the files file

File: debian/polygon2/DEBIAN/md5sums [hunk 2]
```diff
@@ -0,0 +1,9 @@
+98bcd2c2cf5bdbefce8ee5140873c268  usr/lib/python2.7/dist-packages/Polygon/IO.py
```
Comment (gijzelaerr) [inline debian/polygon2/DEBIAN/md5sums]: and you should not commit the DEBIAN folder :)

**Review by gijzelaerr — CHANGES_REQUESTED** [review]
you should not commit the intermediate Debian build files and folders (`DEBIAN/`, `polygon2/`, `tmp/`, `files`, `*.log`)

File: debian/polygon2/DEBIAN/md5sums (hunk 2 above)
Comment (Athanaseus) [reply debian/polygon2/DEBIAN/md5sums]: Duly

File: debian/files (hunk 1 above)
Comment (Athanaseus) [reply debian/files]: Duly

**Pull request merged**